source/CILobbyMenu.h

```diff
@@ -79,7 +79,7 @@ class LobbyMenu {
     std::shared_ptr<cugl::scene2::Button> _winMassBtn;
     std::shared_ptr<cugl::scene2::Label> _winMassLabel;
     std::shared_ptr<cugl::scene2::SceneNode> _winMassBtnLabel;
-    const uint16_t _winMass[5] = { 100, 150, 200, 250, 300 };
+    const uint16_t _winMass[5] = { 4, 5, 6, 7, 8 };
```

Comment:
Can we change the name of this variable since it no longer represents the win mass? Maybe something like _layerSize?

Reply:
Thanks for the feedback, addressed.